Tests/Request/RequestBodyParamConverterTest.php

```diff
@@ -205,6 +205,9 @@ public function testApplyWithSerializerContextOptionsForSymfonySerializer()
         $this->converter->apply($request, $config);
     }
 
+    /**
+     * @group legacy
```

Comment:
this does not make sense either. The codebase already sypports using the new validator. You should just detect the availability of the new validator interface and then use it for the mock instead of the old one.